floss/language/go/extract.py

```diff
@@ -570,6 +571,9 @@ def get_string_blob_strings(pe: pefile.PE, min_length) -> Iterable[StaticString]
     with floss.utils.timing("find struct string candidates"):
         struct_strings = list(sorted(set(get_struct_string_candidates(pe)), key=lambda s: s.address))
 
+    if struct_strings == []:
+        raise ValueError("no struct string candidates found")
```

Comment:
this is still not caught/handled

Reply:
Could you please take a look at the latest commit? I made some updates to address the issue. If you still notice any unresolved concerns, I would greatly appreciate your suggestions on how to handle it properly. Thanks😄

Reply:
we need to catch the exception or return and empty tuple and see if that works

what happens otherwise when run on a binary where the string blob is not found?

Reply:
> what happens otherwise when run on a binary where the string blob is not found?

![image](https://github.com/mandiant/flare-floss/assets/94680887/363113e3-22c3-465c-9ab8-be05a3cae3df)

Reply:
exactly, we want to handle this nicely